cfgov/v1/migrations/0045_auto_20160210_1530.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import wagtail.wagtailcore.fields
+import wagtail.wagtailsnippets.blocks
+import v1.models.snippets
+import wagtail.wagtailcore.blocks
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('v1', '0044_auto_20160208_1703'),
+    ]
+
+    operations = [
```

Comment:
You don't need to break on every `,` maybe just the top level `(),`, just so it's easier to scan through for mistakes.